css/style.css

```diff
@@ -67,6 +67,10 @@ img {
    font-size: 1.5rem;
    padding: 0.938rem;
    order: 2;
+   position: absolute;
```

Comment:
```css
.logo { transform: translateX(-50%);}
```

Reply:
Ohh sorry that's my fault. I'll fix it asap.

P.S. Thanks for the help (It's my first time to Open source contribution) :)